crates/sui-framework/packages/sui-framework/tests/crypto/nitro_attestation_test.move

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[test_only]
+module sui::nitro_attestation_tests {
+    use sui::nitro_attestation;
+    use sui::test_scenario;
+
+    #[test]
+    fun test_nitro_attestation() {
```

Comment:
once we stabilize the move interface, let's test also:
- old cert
- invalid structs
- invalid cert sig

Reply:
a related question, do you think the tests should exist twice in both move and sui-type? or we should port all sui-types tests here? agreed to add more tests. it will be quite involved (i need a mock signer in cert) so good to do it in a separate pr

Reply:
I see, if we cover those cases in the Rust tests then indeed it's enough. Let's only test here the flows of the native call, i.e.,:
- parsing fails (with a hardcoded string)
- parsing succeeds, verification fails 
- everything is good

Reply:
added!